### PR TITLE
Add operator id on CoinATMRadar request

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
Milestone: 7.2 Bullish Bunene

**Comment by RafaelTaranto** [comment]
[Fix CoinATMRadar API](https://trello.com/c/pqp3s3Cg/635-fix-coinatmradar-api)

File: lib/coinatmradar/coinatmradar.js [hunk 1]
```diff
@@ -147,3 +151,7 @@ function update (info) {
     .then(sendRadar)
     .catch(err => logger.error(err))
 }
+
+function computeOperatorId (masterSeed) {
+  return hkdf(masterSeed, 32, {salt: 'lamassu-server-salt', info: 'operator-id'}).toString('hex')
+}
```
Comment (joshmh) [inline lib/coinatmradar/coinatmradar.js]: Configure your editor to require blank lines at EOF.

**Review by joshmh — APPROVED** [review]

File: lib/coinatmradar/coinatmradar.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/coinatmradar/coinatmradar.js]: Oops, I thought that was up. I'm going to configure stardand to run on npm test while adding the elm-admin to l-s.

**Pull request merged**